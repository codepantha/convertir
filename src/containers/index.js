export { default as Currency } from './currency/Currency';
export { default as Converter } from './converter/Converter';
