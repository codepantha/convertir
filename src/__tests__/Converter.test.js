import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import { Converter } from '../containers';

it('renders correctly', () => {
  const tree = renderer
    .create(<Provider store={store}><Converter /></Provider>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
