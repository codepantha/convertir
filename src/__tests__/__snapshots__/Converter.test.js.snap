// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="convertir__converter"
>
  <h2
    className="section__title"
  >
    Use our free converter
  </h2>
  <div
    className="convertir__converter-from flex items-center"
  >
    <p
      className="f4"
    >
      
    </p>
    <input
      className="pa1 br3 ba outline-0 b--black-0125"
      onChange={[Function]}
      placeholder="1"
      type="text"
      value={1}
    />
    <svg
      fill="none"
      height="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 24 24"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M12.0519 14.8285L13.4661 16.2427L17.7088 12L13.4661 7.7574L12.0519 9.17161L13.8804 11H6.34321V13H13.8803L12.0519 14.8285Z"
        fill="currentColor"
      />
      <path
        clipRule="evenodd"
        d="M19.7782 19.7782C24.0739 15.4824 24.0739 8.51759 19.7782 4.22183C15.4824 -0.0739417 8.51759 -0.0739417 4.22183 4.22183C-0.0739417 8.51759 -0.0739417 15.4824 4.22183 19.7782C8.51759 24.0739 15.4824 24.0739 19.7782 19.7782ZM18.364 18.364C21.8787 14.8492 21.8787 9.15076 18.364 5.63604C14.8492 2.12132 9.15076 2.12132 5.63604 5.63604C2.12132 9.15076 2.12132 14.8492 5.63604 18.364C9.15076 21.8787 14.8492 21.8787 18.364 18.364Z"
        fill="currentColor"
        fillRule="evenodd"
      />
    </svg>
  </div>
  <div
    className="convertir__converter-to flex shadow-5 items-center"
  >
    <p
      className="f4"
    />
    <p
      className="f3"
    >
      NaN
    </p>
    <svg
      fill="none"
      height="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 24 24"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M12.0519 14.8285L13.4661 16.2427L17.7088 12L13.4661 7.7574L12.0519 9.17161L13.8804 11H6.34321V13H13.8803L12.0519 14.8285Z"
        fill="currentColor"
      />
      <path
        clipRule="evenodd"
        d="M19.7782 19.7782C24.0739 15.4824 24.0739 8.51759 19.7782 4.22183C15.4824 -0.0739417 8.51759 -0.0739417 4.22183 4.22183C-0.0739417 8.51759 -0.0739417 15.4824 4.22183 19.7782C8.51759 24.0739 15.4824 24.0739 19.7782 19.7782ZM18.364 18.364C21.8787 14.8492 21.8787 9.15076 18.364 5.63604C14.8492 2.12132 9.15076 2.12132 5.63604 5.63604C2.12132 9.15076 2.12132 14.8492 5.63604 18.364C9.15076 21.8787 14.8492 21.8787 18.364 18.364Z"
        fill="currentColor"
        fillRule="evenodd"
      />
    </svg>
  </div>
</div>
`;
